newIDE/app/src/ResourcesList/index.js

```diff
@@ -173,7 +172,13 @@ export default class ResourcesList extends React.Component<Props, State> {
       {
         label: 'Scan for Images',
         click: () => {
-          this._scanForNewResources();
+          this._scanForNewResources(IMAGE_EXTENSIONS, () => new gd.ImageResource());
+        },
+      },
+      {
+        label: 'Scan for Audio',
+        click: () => {
+          this._scanForNewResources(AUDIO_EXTENSIONS, () => new gd.AudioResource());
```

Comment:
Sweet, you might want to merge master into your branch and benefit from functions to scan for unused audio and fonts too :)

Reply:
Are you sure you added that to the editor? I merged it and the menu has no option to scan for text or remove unused audio.
I can add the options if you want to :)

Reply:
Ok I added the commands now  :)

Reply:
Yeah I mean I added the function to libGD.js but not in the editor to avoid create conflicts with your PR :)

Reply:
Merged and updated to include the new scan/remove commands

Reply:
Looks good :)